helix-term/src/ui/lsp.rs

```diff
@@ -155,10 +155,7 @@ impl Component for SignatureHelp {
 
         let sig = &self.signatures[self.active_signature];
 
-        if PADDING >= viewport.1 || PADDING >= viewport.0 {
-            return None;
-        }
```

Comment:
Was this the bug? Returning `None` instead of a default size?

Reply:
Yeah this was an existing bug that caused crashes. required_size is only meant to always return None (for components that can't be embedded in a popup) or always meant to return Some. The API is a bit confusing and was very inconsistently used so this seems to have slipped trough at some point.